test/testmathlib.cpp

```diff
@@ -364,11 +364,13 @@ class TestMathLib : public TestFixture {
             ASSERT_EQUALS(u, MathLib::toULongNumber("0xFFFFFFFFFFFFFFFF"));
         }
         {
+#pragma warning (suppress : 4146) // unary minus operator applied to unsigned type
```

Comment:
could we disable this warning in the test project? some strange code can be allowed in test code.